apis/v1alpha1/tlsroute_types.go

```diff
@@ -65,11 +65,29 @@ type TLSRouteStatus struct {
 
 // TLSRouteRule is the configuration for a given rule.
 type TLSRouteRule struct {
-	// Matches define conditions used for matching the rule against an
-	// incoming TLS handshake. Each match is independent, i.e. this
-	// rule will be matched if **any** one of the matches is satisfied.
-	// If unspecified, all requests from the associated gateway TLS
-	// listener will match.
+	// Matches define conditions used for matching the rule against incoming TLS
+	// connections. Each match is independent, i.e. this rule will be matched if
+	// **any** one of the matches is satisfied. If unspecified, all requests
```

Comment:
See my comment above:

If unspecified (i.e. empty), this Rule will match all requests for the associated Listener.

Reply:
Updated, thanks!